packages/opds-web-client/src/components/Root.tsx

```diff
@@ -138,14 +138,13 @@ export class Root extends React.Component<RootProps, RootState> {
     );
 
     let showCollection = this.props.collectionData && !this.props.bookData;
-    let showBook = this.props.bookData;
+    const showBook = !!this.props.bookData;
```

Comment:
This was updated but on line 265 it was replaced by `this.props.bookData`. This is still being used for `showCollectionContainer` on line 150. Shouldn't that be updated as well?

Reply:
We can replace it and get rid of `showBook`, but it's not necessary. The reason it was necessary to replace `showBook` in that instance was because the following code then depended on `this.props.bookData` being of type `BookData`, and typescript would error unless it could narrow the type to that... whereas in this case the `showCollectionContainer` is used to control the display of some UI which doesn't depend on `this.props.bookData`, so there is no typescript error. In the end, the functionality of the code is the same whether you use `showBook` or `this.props.bookData`, it's just that typescript can't be sure that `this.props.bookData` is actually defined if you use `showBook`... does that make any sense?